shopinvader/services/customer.py

```diff
@@ -35,6 +38,24 @@ def create(self, **params):
         self._post_create(self.work.partner)
         return self._prepare_create_response(binding)
 
+    def update(self, _id, **params):
+        # We get the binding from the partner
+        if not self.partner:
+            raise UserError(_("You must provide a partner"))
+
+        partner = self._get(_id)
+        if not partner.shopinvader_bind_ids:
+            raise UserError(_("Partner %s is not a customer") % partner.id)
+        binding = partner.shopinvader_bind_ids[0]
+
+        vals = self._prepare_params(params, mode="update")
+        binding.write(vals)
+        self._load_partner_work_context(binding)
```

Comment:
```suggestion
```
If is the owner of the partner to call this endpoint you don't need to load the partner ctx.
The partner ctx it's already set at this stage.
Moreover, if the user calling this endpoint is not the owner, you don't want to swap its ctx IMO.
Loading the ctx is required only when you create because there's no ctx yet.